modules/ROOT/pages/munit-test-concept.adoc

```diff
@@ -79,6 +79,26 @@ Valid values for `ignore` are true and false. If the attribute is not present, t
 .In Anypoint Studio, you can check the Ignore Test option.
 image::munit-test-concept-de4c9.png[ignore-test]
 
+=== Using expressions to ignore a test
+
+Since MUnit 2.2, the `ignore` attribute allows expression to be used. This allows you to ignore a test based on the
```

Comment:
Since this is the MUnit 2.2 documentation,  I don't think it's necessary to add the minimum version where this is supported. 
Best is to not compare this with previous versions and just document it as another feature of this version.